tests/syft/lib/numpy/array_test.py

```diff
@@ -0,0 +1,37 @@
+# third party
+import numpy as np
```

Comment:
Make sure to move the import numpy as np and other np calls to inside the test.
That way when the test collection process checks for `numpy` and if it doesn't find it it wont crash collecting the test file. In reality no one will not have numpy but its still how the vendor lib tests are done.

Reply:
Thank you!